awx/main/management/commands/inventory_import.py

```diff
@@ -95,28 +95,9 @@ def build_env(self):
 
     def get_base_args(self):
         # get ansible-inventory absolute path for running in bubblewrap/proot, in Popen
-        for path in os.environ["PATH"].split(os.pathsep):
-            potential_path = os.path.join(path.strip('"'), 'ansible-inventory')
-            if os.path.isfile(potential_path) and os.access(potential_path, os.X_OK):
-                logger.debug('Using system install of ansible-inventory CLI: {}'.format(potential_path))
-                return [potential_path, '-i', self.source]
-
-        # Stopgap solution for group_vars, do not use backported module for official
-        # vendored cloud modules or custom scripts TODO: remove after Ansible 2.3 deprecation
-        if self.is_vendored_source or self.is_custom:
-            self.method = 'inventory script invocation'
-            return [self.source]
-
-        # ansible-inventory was not found, look for backported module TODO: remove after Ansible 2.3 deprecation
-        abs_module_path = os.path.abspath(os.path.join(
-            os.path.dirname(__file__), '..', '..', '..', 'plugins',
-            'ansible_inventory', 'backport.py'))
-        self.method = 'ansible-inventory backport'
-
-        if not os.path.exists(abs_module_path):
-            raise ImproperlyConfigured('Cannot find inventory module')
-        logger.debug('Using backported ansible-inventory module: {}'.format(abs_module_path))
-        return [abs_module_path, '-i', self.source]
+        bargs= ['ansible-inventory', '-i', self.source]
```

Comment:
Changing from `abs_module_path` to `'ansible-inventory'` is one of the few changes that I'm a little unsure of. The logic is that searching PATH for the executable is totally unnecessary. However, we might either need to pull the logic back in, or do `which ansible-inventory` as another subprocess because of some issues encountered with inventory plugins. So that remains TBD, but given this code state, I would still argue that searching PATH is strictly unnecessary.

Reply:
```
$ python3
Python 3.6.5 (default, Apr 25 2018, 14:23:58) 
[GCC 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.1)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from shutil import which
>>> which('ansible-inventory')
'/Users/alancoding/.virtualenvs/tower_docker/bin/ansible-inventory'
```

@ryanpetrello this requires python3, but that shouldn't be a problem for long 🎉 . I am 100% sold on this option.